employee-service/src/test/java/com/rabbit/samples/employeeservice/unittests/services/EmployeeServiceUnitTest.java

```diff
@@ -4,74 +4,52 @@
+import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.annotation.Import;
 import org.springframework.test.annotation.IfProfileValue;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.BDDMockito.*;
 
-@IfProfileValue(name = "spring.testing.profile", values = {"all", "unit"})
-@RunWith(SpringRunner.class)
-public class EmployeeServiceUnitTest {
-
-	/*
-		To check the Service class, we need to have an instance of Service class created and available as a @Bean so that we can @Autowire it in our test class.
-		This configuration is achieved by using the @TestConfiguration annotation.
 
-		PLEASE NOTE:
-		During component scanning, we might find components or configurations created only for specific tests accidentally get picked up everywhere.
-		To help prevent that, Spring Boot provides @TestConfiguration annotation that can be used on classes in src/test/java to indicate that they should
-		not be picked up by scanning.
-	 */
-	@TestConfiguration
-	@Import(EmployeeServiceImpl.class)
-	static class EmployeeServiceUnitTestContextConfiguration {
-
-		// no-op
-	}
-
-
-	@Autowired
-	private EmployeeService employeeService;
+@RunWith(MockitoJUnitRunner.class)
+public class EmployeeServiceUnitTest {
 
-	/*
-		Mocking support provided by Spring Boot Test: it creates a Mock for the EmployeeRepository which can be used to bypass the call to the actual
-		EmployeeRepository.
+	@InjectMocks
+	private EmployeeServiceImpl employeeService;
 
-		PLEASE NOTE:
-		In the EmployeeService is specified a @Qualifier for the EmployeeRepository injection, that refers to a bean named "employeeRepository".
-		During the test execution, the EmployeeRepository bean is mocked with a name different than the variable name (something like
-		com.rabbit.samples.employeeservice.unittests.services.EmployeeRepository#0), leading to miss the EmployeeService injection.
-	 */
-	@MockBean(name = "employeeRepository")
+	@Mock
 	private EmployeeRepository employeeRepository;
 
 	final String name = "alex";
 
 	@Test
-	public void givenValidName_thenReturnFoundEmployee() throws Exception {
+	public void givenValidName_thenReturnFoundEmployee() {
 
 		// given
-		BDDMockito
-				.when(employeeRepository.findByName(name))
+		when(employeeRepository.findByName(name))
 				.thenReturn(Employee.builder().name(name).build());
 
 		// when
 		final Employee found = employeeService.getByName(name);
 
 		// then
-		Assert.assertEquals(name, found.getName());
+		assertThat(name).isEqualTo(found.getName());
 
 		// verify
-		BDDMockito.verify(employeeRepository, VerificationModeFactory.times(1)).findByName(name);
-		BDDMockito.reset(employeeRepository);
```

Comment:
I would keep this line in order to avoid confusing between different tests.

Reply:
here is a good example of good use of static import. The Hamcrest matcher would read better 

verify(employeeRepository, times(1) )

what I care about is that the method is called once, VerificationModeFactory does not give us anything useful.